src/routes/(inner)/docs/colors/+page.svelte

```diff
@@ -18,7 +18,7 @@
 				'[style]-success-[50-900]',
 				'Used for successful or positive actions, such as a form submit alert.'
 			],
-			[`<div class="${cSwatch} bg-warning-500" />`, '[style]-warning-[50-900]', 'Used for negative or harmful actions, such as errors.'],
+			[`<div class="${cSwatch} bg-warning-500" />`, '[style]-warning-[50-900]', 'Used for negative or harmful actions, such as warnings or errors.'],
```

Comment:
Not sure we should leave `or errors` in there when there is a separate class called error.